src/styles/_variables.js

```diff
@@ -8,12 +8,13 @@ export const //colors
   $pinkColor = '#DE00FF',
   $blueColor = '#047CFF',
   $whiteColor = '#fafafa',
-  $whiteColorSemi = '	rgba(250,250,250,.2)'
+  $whiteColorSemi = '	rgba(250,250,250,.2)',
```

Comment:
silly question,,, this is a space between single quotes and rgba `$whiteColorSemi = '    RGBA`

Reply:
just a typo probably. fixed it.